### PR TITLE
fix: change clap version to env!("CARGO_PKG_VERSION")

### PR DESCRIPTION
**Pull request by LycrusHamster**
**What type of PR is this?**

 fix: change clap version to env!("CARGO_PKG_VERSION") and fix huobi-chain version

**What this PR does / why we need it**:

now clap will response -V / --version as same as cargo.toml/package/version

### CODE REVIEW

File: services/asset/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "asset"
-version = "0.4.0-beta2"
+version = "0.4.0-Alpha"
```
Comment (homura) [inline services/asset/Cargo.toml:3]: It's rare to see someone mark a prerelease with an `Alpha`, `alpha` might be more appropriate

Comment (yejiayu) [reply services/asset/Cargo.toml:3]: alpha.0

**Pull request closed without merging**